src/pages/scripts/tutorials/script-result/index.md

```diff
@@ -1,50 +1,49 @@
 # Setting Script Result
 
-InDesign now has the functionality to "set result" of a UXP script. After execution, a script can have its own result which then can be used for debugging purposes on InDesign Server. The result of the script can also be stored in a variable and be used by another script in both InDesign App and InDesign Server. Setting the script result can be accomplished using the script.`setResult()` API, the sections below demonstrate the various use cases. 
+Since InDesign v18.4, you can "set result" of a UXP script. 
+
+After execution, a script can have its own result which then can be used for debugging purposes on InDesign Server. The result of the script can also be stored in a variable and be used by another script in both InDesign App and InDesign Server. Setting the script result can be accomplished using the `script.setResult()` API.
```

Comment:
@padmkris123 I have reworded the para a bit, please check if appropriate.

**After execution, a script can return a result, which could be used for evaluating the outcome of script execution.  The result is set using `script.setResult()` API as demonstrated below.**